pkg/helpers/client.go

```diff
@@ -30,7 +32,7 @@ func GetAPIServer(kubeClient kubernetes.Interface) (string, error) {
 	return cluster.Server, nil
 }
 
-func GetCACert(kubeClient kubernetes.Interface) ([]byte, error) {
+func GetCACert(kubeClient kubernetes.Interface, useBootstrapToken bool) ([]byte, error) {
```

Comment:
do you need useBootstrapToken?

Reply:
You are correct it is not needed anymore